src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_ITupleExpression.vb

```diff
@@ -0,0 +1,330 @@
+]]>.Value
+
+            Dim expectedDiagnostics = String.Empty
+
+            VerifyOperationTreeAndDiagnosticsForTest(Of TupleExpressionSyntax)(source, expectedOperationTree, expectedDiagnostics)
+        End Sub
+
+        <Fact, WorkItem(10856, "https://github.com/dotnet/roslyn/issues/10856")>
+        Public Sub TupleExpression_NamedArguments()
+            Dim source = <![CDATA[
+Option Strict Off
+Imports System
+
+Class C
+    Shared Sub Main()
+        Dim t = (A:=1, B:=2)'BIND:"(A:=1, B:=2)"
+    End Sub
+End Class]]>.Value
+
+            Dim expectedOperationTree = <![CDATA[
+ITupleExpression (OperationKind.TupleExpression, Type: (A As System.Int32, B As System.Int32)) (Syntax: '(A:=1, B:=2)')
+  Elements(2): ILiteralExpression (Text: 1) (OperationKind.LiteralExpression, Type: System.Int32, Constant: 1) (Syntax: '1')
+    ILiteralExpression (Text: 2) (OperationKind.LiteralExpression, Type: System.Int32, Constant: 2) (Syntax: '2')
+]]>.Value
+
+            Dim expectedDiagnostics = String.Empty
+
+            VerifyOperationTreeAndDiagnosticsForTest(Of TupleExpressionSyntax)(source, expectedOperationTree, expectedDiagnostics)
+        End Sub
+
+        <Fact, WorkItem(10856, "https://github.com/dotnet/roslyn/issues/10856")>
+        Public Sub TupleExpression_NamedElementsInTupleType()
+            Dim source = <![CDATA[
+Option Strict Off
+Imports System
+
+Class C
+    Shared Sub Main()
+        Dim t As (A As Integer, B As Integer) = (1, 2)'BIND:"(1, 2)"
+    End Sub
+End Class]]>.Value
+
+            Dim expectedOperationTree = <![CDATA[
+IConversionExpression (ConversionKind.Basic, Implicit) (OperationKind.ConversionExpression, Type: (A As System.Int32, B As System.Int32)) (Syntax: '(1, 2)')
+  ITupleExpression (OperationKind.TupleExpression, Type: (System.Int32, System.Int32)) (Syntax: '(1, 2)')
+    Elements(2): ILiteralExpression (Text: 1) (OperationKind.LiteralExpression, Type: System.Int32, Constant: 1) (Syntax: '1')
+      ILiteralExpression (Text: 2) (OperationKind.LiteralExpression, Type: System.Int32, Constant: 2) (Syntax: '2')
+]]>.Value
+
+            Dim expectedDiagnostics = String.Empty
+
+            VerifyOperationTreeAndDiagnosticsForTest(Of TupleExpressionSyntax)(source, expectedOperationTree, expectedDiagnostics)
+        End Sub
+
+        <Fact, WorkItem(10856, "https://github.com/dotnet/roslyn/issues/10856")>
+        Public Sub TupleExpression_NamedElementsAndImplicitConversions()
+            Dim source = <![CDATA[
+Option Strict Off
+Imports System
+
+Class C
+    Shared Sub Main()
+        Dim t As (A As Int16, B As String) = (A:=1, B:=Nothing)'BIND:"(A:=1, B:=Nothing)"
+    End Sub
+End Class]]>.Value
+
+            Dim expectedOperationTree = <![CDATA[
+IConversionExpression (ConversionKind.Basic, Implicit) (OperationKind.ConversionExpression, Type: (A As System.Int16, B As System.String)) (Syntax: '(A:=1, B:=Nothing)')
```

Comment:
The conversion feels unexpected given the syntax we examine. Also, there is no type change from the operand.